docs/tutorials/running_r.md

```diff
@@ -18,3 +18,7 @@ VSCode window.
 send code from the `.R` file to the R terminal by pressing `cmd/ctrl + enter`.
 
 ![alt text](../assets/rdev12.png) ![alt text](../assets/rdev5.png)
+
+```Rconsole
+hist(rnorm(1000))
+```
```

Comment:
Rconsole blocks are for displaying input and output in an R session, similar to console blocks for sessions in a bash terminal. You need to include the prompt for input lines so that syntax highlighting is applied, i.e.

```Rconsole
> hist(rnorm(1000))
```

However, an Rconsole block is not suitable in the case. The screenshot is intended to show the R script file, the `hist(rnorm(1000))` command sent to the R terminal and the histogram in the plot window. Unfortunately, the screenshot here is wrong, it shows `source("/workspaces/r-dev-env/R/test.R", encoding = "UTF-8")`, which is the result of running `Cmd/Ctrl + Shift + s` not `Cmd/Ctrl + enter`, which is the short-cut we wish to document. You could fix this at the same time as the code fencing or open a separate issue about this.